app/build.gradle

```diff
@@ -120,7 +122,7 @@ dependencies {
         transitive = false
     }
     annotationProcessor 'androidx.annotation:annotation:1.1.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.10.0'
+    annotationProcessor "com.github.bumptech.glide:compiler:${glideVersion}"
```

Comment:
This causes the build to fail.

Reply:
Oops, looks like that still slipped in. Perf comparison is on my todo list but I've been a bit busy with other work :)